w3-account.md

```diff
@@ -0,0 +1,224 @@
+# User Accounts
+
+![status:wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+[![hackmd-github-sync-badge](https://hackmd.io/8NywALT8Qp-cf0MSugZMDw/badge)](https://hackmd.io/8NywALT8Qp-cf0MSugZMDw)
+
+## Editors
+
+- [Irakli Gozalishvili](https://github.com/Gozala), [DAG House](https://dag.house/)
+
+## Authors
+
+- [Irakli Gozalishvili](https://github.com/Gozala), [DAG House](https://dag.house/)
+
+
+# Abstract
+
+
+In web3.storage we describe concept of an account as convenience for aggregating and managing capabilities across various user spaces under same identity, simplifying recovery and authorization flows.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+
+# Introduction
+
+## Motivation
+
+In web3.storage users MAY create number of user spaces simply by generating asymmetric key pair. They MAY be delegated capabilities for other user spaces. Managing these delegations and keypairs across multiple spaces, agents and devices can get complicated.
+
+To address this we propose a concept of an account, which is a [principal][] that can be delegated all relevant capabilities across various user spaces.
+
+We also propose account authorization flow that would allowing allow authorized agent to act on behalf of the account [principal][].
+
+
+# Terminology
+
+## Account
+
+User account is a [principal][] identified by [`did:mailto`][] (listed in a UCAN's `iss` or `aud` field).
```

Comment:
> Why have this be did:mailto: and not mailto: ?

Because principals in UCAN spec are DIDs, if they were arbitrary URIs than using just `mailto:` URI would have made a lot more sense. Perhaps we can work towards allowing UCANs to use arbitrary URIs in principals, but I would pursue that separately and then update to just `mailto:` as opposed to getting blocked on that.

> If we do really want to define/use a did:mailto, how would did resolution work (e.g. [here is how did:dns resolution works](https://danubetech.github.io/did-method-dns/#resolve))

My plan had been do spec that out here https://github.com/ucan-wg/did-mailto, but I don't have bandwidth for that right now.

Reply:
> how to create, read (resolve to did doc), update, and delete dids using that method.

In nutshell, resolution schema is you send them an email asking what's their DID document and they respond you back with did:key corresponding to their document